src/datanode/src/store.rs

```diff
@@ -138,35 +136,38 @@ pub(crate) fn clean_temp_dir(dir: &str) -> Result<()> {
     Ok(())
 }
 
+/// FIXME: we need to use reqwest 0.12 here.
 pub(crate) fn build_http_client() -> Result<HttpClient> {
-    let http_builder = {
-        let mut builder = reqwest::ClientBuilder::new();
-
-        // Pool max idle per host controls connection pool size.
-        // Default to no limit, set to `0` for disable it.
-        let pool_max_idle_per_host = env::var("_GREPTIMEDB_HTTP_POOL_MAX_IDLE_PER_HOST")
-            .ok()
-            .and_then(|v| v.parse::<usize>().ok())
-            .unwrap_or(usize::MAX);
-        builder = builder.pool_max_idle_per_host(pool_max_idle_per_host);
-
-        // Connect timeout default to 30s.
-        let connect_timeout = env::var("_GREPTIMEDB_HTTP_CONNECT_TIMEOUT")
-            .ok()
-            .and_then(|v| v.parse::<u64>().ok())
-            .unwrap_or(30);
-        builder = builder.connect_timeout(Duration::from_secs(connect_timeout));
-
-        // Pool connection idle timeout default to 90s.
-        let idle_timeout = env::var("_GREPTIMEDB_HTTP_POOL_IDLE_TIMEOUT")
-            .ok()
-            .and_then(|v| v.parse::<u64>().ok())
-            .unwrap_or(90);
-
-        builder = builder.pool_idle_timeout(Duration::from_secs(idle_timeout));
-
-        builder
-    };
-
-    HttpClient::build(http_builder).context(error::InitBackendSnafu)
+    // let http_builder = {
+    //     let mut builder = reqwest::ClientBuilder::new();
+    //
+    //     // Pool max idle per host controls connection pool size.
+    //     // Default to no limit, set to `0` for disable it.
+    //     let pool_max_idle_per_host = env::var("_GREPTIMEDB_HTTP_POOL_MAX_IDLE_PER_HOST")
+    //         .ok()
+    //         .and_then(|v| v.parse::<usize>().ok())
+    //         .unwrap_or(usize::MAX);
+    //     builder = builder.pool_max_idle_per_host(pool_max_idle_per_host);
+    //
+    //     // Connect timeout default to 30s.
+    //     let connect_timeout = env::var("_GREPTIMEDB_HTTP_CONNECT_TIMEOUT")
+    //         .ok()
+    //         .and_then(|v| v.parse::<u64>().ok())
+    //         .unwrap_or(30);
+    //     builder = builder.connect_timeout(Duration::from_secs(connect_timeout));
+    //
+    //     // Pool connection idle timeout default to 90s.
+    //     let idle_timeout = env::var("_GREPTIMEDB_HTTP_POOL_IDLE_TIMEOUT")
+    //         .ok()
+    //         .and_then(|v| v.parse::<u64>().ok())
+    //         .unwrap_or(90);
+    //
+    //     builder = builder.pool_idle_timeout(Duration::from_secs(idle_timeout));
+    //
+    //     builder
+    // };
+    //
+    // HttpClient::build(http_builder).context(error::InitBackendSnafu)
```

Comment:
cc code owner @killme2008

Reply:
This seems a regression. cc @Xuanwo 

I'll take a look also.

Reply:
I left a `FIXME` here, it's not a regression.

Reply:
Let me try to upgrade it or use `reqwest_012 = { package = "reqwest", version = "0.12" }` here.

Reply:
Resolved at https://github.com/GreptimeTeam/greptimedb/pull/4037/commits/ad0793b48cb0ac19fbd7c52be215140d96ca6eb5.